### PR TITLE
Update README.md and WORKING-GROUP.md

### PR DESCRIPTION
**Pull request by Andersson007**
Update README.md and WORKING-GROUP.md

Please suggest things to add, change or remove

### CODE REVIEW

File: WORKING-GROUPS.md [hunk 1]
```diff
@@ -2,25 +2,26 @@
...
+* Group name and charter (why the group exists);
+* Registered [real-time chat](https://docs.ansible.com/ansible/devel/community/communication.html#real-time-chat) channel;
+* Group of users (at least two!) who will be driving the agenda of the working group;
 * Dedicated wiki space.
 
 The basic responsibilities of a working group:
-* Be responsive on your IRC channel;
+* Be responsive on your real-time chat channel;
 * Report semi-regularly on the cool stuff that your working group is working on;
 * Keep your wiki space clean!
```
Comment (anweshadas) [inline WORKING-GROUPS.md:16]: Can you also please include the following?

```
* Make sure to follow the [Ansible Community Code of Conduct (https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
```

File: README.md [hunk 2]
```diff
@@ -10,32 +10,36 @@ This repository serves as a space for the Ansible Community Working Group to man
...
-## What We're Working On
+## What we are working on
 
-* [Community Project](https://github.com/ansible/community/projects/1)
-* [Contributor Experience Project](https://github.com/orgs/ansible/projects/2)
+* [Community Topics](https://github.com/ansible-community/community-topics)
+* [Collection Inclusion](https://github.com/ansible-collections/ansible-inclusion)
+* [Ansible Documentation](https://github.com/orgs/ansible/projects/94/views/1)
+* [Ansible Collections](https://github.com/ansible-collections)
+* [Projects of Ansible Ecosystem](https://github.com/ansible)
```
Comment (oraNod) [inline README.md:35]: What about "Ecosystem of Ansible Projects" or just "Projects in the Ansible Ecosystem"

File: WORKING-GROUPS.md (hunk 1 above)
Comment (Andersson007) [reply WORKING-GROUPS.md:17]: Makes sense to mention here, thanks, done

File: README.md [hunk 3]
```diff
@@ -10,32 +10,36 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Our Goals
 
-* Encourage transparency, collaboration, and accountability through tooling and lightweight project tracking and/or management
-* Provide guidance and advice to other working groups in the Ansible community
-* Facilitate the creation of new initiatives and Working Groups
+* Encourage transparency, collaboration, and accountability through tooling and lightweight project tracking and/or management.
+* Provide guidance and advice to other working groups in the Ansible community.
```
Comment (oraNod) [inline README.md:14]: What other working groups are there?

Also I'd prefer something along the lines of "Assist other working groups in the Ansible community for our mutual success." IMO "provide guidance and advice" sounds a little too much like we're governing other communities.

File: README.md [hunk 4]
```diff
@@ -10,32 +10,36 @@ This repository serves as a space for the Ansible Community Working Group to man
...
+* [Bullhorn](https://github.com/ansible/community/wiki/News#the-bullhorn): Ansible contributor newsletter.
+* [Collection Bot](https://github.com/ansible-community/collection_bot): Automation of GitHub issue/pull request processing for ansible collections.
 
 ## Groups
 
-**NOTE:** For the majority of IRC channels you must [register your nick with irc.libera.chat](https://libera.chat/guides/registration)
-
 * [All Ansible Working Groups](https://github.com/ansible/community/wiki)
-* [All Ansible Email lists, IRC and Working Groups](https://docs.ansible.com/ansible/devel/community/communication.html)
+* [All Ansible Email lists, Matrix/IRC channels and Working Groups](https://docs.ansible.com/ansible/devel/community/communication.html)
```
Comment (oraNod) [inline README.md:27]:
```suggestion
* [All Ansible email lists, Matrix/IRC channels and working groups](https://docs.ansible.com/ansible/devel/community/communication.html)
```

File: WORKING-GROUPS.md [hunk 5]
```diff
@@ -2,25 +2,27 @@
...
 The basic responsibilities of a working group:
-* Be responsive on your IRC channel;
+* Follow the [Ansible Community Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
+* Be responsive on your real-time chat channel;
 * Report semi-regularly on the cool stuff that your working group is working on;
 * Keep your wiki space clean!
 
+You can find the list of existing working groups in the [Ansible communication guide](https://docs.ansible.com/ansible/devel/community/communication.html#working-groups).
+
 ## Requesting a Working Group
```
Comment (oraNod) [inline WORKING-GROUPS.md:21]:
```suggestion
## Requesting a working group
```

File: README.md [hunk 6]
```diff
@@ -10,32 +10,36 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Our Goals
 
-* Encourage transparency, collaboration, and accountability through tooling and lightweight project tracking and/or management
-* Provide guidance and advice to other working groups in the Ansible community
-* Facilitate the creation of new initiatives and Working Groups
+* Encourage transparency, collaboration, and accountability through tooling and lightweight project tracking and/or management.
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md:13]: We are free to reformulate or add/remove what we want in these ancient docs:)

File: README.md (hunk 3 above)
Comment (Andersson007) [reply README.md:14]: agreed

Comment (Andersson007) [reply README.md:14]: I'll remove the word "other".

Comment (Andersson007) [reply README.md]: done, PTAL

**Review by oraNod — APPROVED** [review]

File: WORKING-GROUPS.md [hunk 7]
```diff
@@ -33,38 +35,22 @@ WORKING GROUP PURPOSE:
...
-  * IRC: `#ansible-devel`  `#ansible` 
-  * [Changes impacting Contributors](https://github.com/ansible/community/issues/346)
-  * [The Bullhorn](https://github.com/ansible/community/issues/546)
-  * Google Groups: `ansible-project` & `ansible-devel`
+## Need help?
 
-## Help!
+If you get stuck or want to know more, join us in one of the following channels:
+* [#community:ansible.com](https://matrix.to/#/#community:ansible.com) on [Matrix](https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-matrix)
+* `#ansible-community` on Libera.Chat [IRC](https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-irc)
```
Comment (anweshadas) [inline WORKING-GROUPS.md:54]: Can you include the following too?

```
Resource to follow
* [Ansible Community Guide](https://docs.ansible.com/ansible/latest/community/index.html) is wonderful resource to answer some your quarries.
```

Comment (Andersson007) [reply WORKING-GROUPS.md:54]: SGTM, done, thanks

**Comment by Andersson007** [comment]
@oraNod @anweshadas thanks for reviewing!

**Pull request merged**